src/server/controllers/review.controller.ts

```diff
@@ -152,7 +153,27 @@ export const toggleReactionHandler = async ({
   }
 };
 
-export const getReviewHandler = async ({ input }: { input: GetByIdInput }) => {
+export type ReviewDetails = AsyncReturnType<typeof getReviewDetails>;
+export const getReviewDetails = async ({ input: { id } }: { input: GetByIdInput }) => {
```

Comment:
Just nitpicking again, but let's add `Handler` suffix to keep consistency
```suggestion
export const getReviewDetailsHandler = async ({ input: { id } }: { input: GetByIdInput }) => {
```